x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engine/engine_schema.tsx

```diff
@@ -5,43 +5,274 @@
 import { EnterpriseSearchEnginesPageTemplate } from '../layout/engines_page_template';
 
 import { EngineIndicesLogic } from './engine_indices_logic';
 
 import { EngineViewLogic } from './engine_view_logic';
+import './engine_schema.scss';
+
+const CONFLICT = 'conflict';
+const UNMAPPED = 'unmapped';
+
+const SchemaFieldDetails: React.FC<{ schemaField: SchemaField }> = ({ schemaField }) => {
+  const { navigateToUrl } = useValues(KibanaLogic);
+  const notInAllIndices = schemaField.indices.some((i) => i.type === UNMAPPED);
+
+  const columns: Array<EuiBasicTableColumn<SchemaField['indices'][0]>> = [
+    {
+      field: 'name',
+      name: i18n.translate(
+        'xpack.enterpriseSearch.content.engine.schema.fieldIndices.index.columnTitle',
+        {
+          defaultMessage: 'Parent index',
+        }
+      ),
+      render: (name: string) => (
+        <EuiLinkTo
+          to={generateEncodedPath(SEARCH_INDEX_TAB_PATH, {
+            indexName: name,
+            tabId: 'index_mappings',
+          })}
+        >
+          {name}
+        </EuiLinkTo>
+      ),
+    },
+    {
+      field: 'type',
+      name: i18n.translate(
+        'xpack.enterpriseSearch.content.engine.schema.fieldIndices.type.columnTitle',
+        {
+          defaultMessage: 'Field mapped as',
+        }
+      ),
+      render: (name: string) => {
+        if (name === UNMAPPED)
+          return (
+            <EuiBadge color="warning">
+              <FormattedMessage
+                id="xpack.enterpriseSearch.content.engine.schema.fieldIndices.type.unmapped"
+                defaultMessage="Unmapped"
+              />
+            </EuiBadge>
+          );
+        return name;
+      },
+    },
+    {
+      actions: [
+        {
+          description: 'View index mappings',
+          icon: 'eye',
+          name: 'View index',
+          onClick: (item: SchemaField['indices'][0]) => {
+            navigateToUrl(
+              generateEncodedPath(SEARCH_INDEX_TAB_PATH, {
+                indexName: item.name,
+                tabId: 'index_mappings',
+              })
+            );
+          },
+          type: 'icon',
+        },
+      ],
+    },
+  ];
+
+  return (
+    <EuiPanel hasBorder={false} hasShadow={false} paddingSize="l" color="transparent">
+      <EuiFlexGroup direction="column" gutterSize="l">
+        {notInAllIndices && (
+          <EuiCallOut
+            iconType="iInCircle"
+            title={
+              <FormattedMessage
+                id="xpack.enterpriseSearch.content.engine.schema.fieldIndices.notInAllIndices.title"
+                defaultMessage="This field is not mapped in every index."
+              />
+            }
+          >
+            <EuiText size="s">
+              <p>
+                <FormattedMessage
+                  id="xpack.enterpriseSearch.content.engine.schema.fieldIndices.notInAllIndices.description"
+                  defaultMessage="Learn more about field mapping in our documentation."
```

Comment:
Is there a way to link to our docs?

Reply:
yes these is! good catch, thanks. https://github.com/elastic/kibana/pull/153602/commits/ea452fc69c25ba1d5ea5810981fc59ef5c4d056c